linera-views/src/key_value_store_view.rs

```diff
@@ -547,9 +548,7 @@ where
         if self.was_cleared {
             return Ok(None);
         }
-        let iter = self.deleted_prefixes.iter();
-        let mut lower_bound = GreatestLowerBoundIterator::new(0, iter);
-        if !lower_bound.is_index_absent(index) {
+        if !is_index_absent(&self.deleted_prefixes, index) {
```

Comment:
Maybe it would be better to implement `is_index_present` instead, so we don't have double negations.

Reply:
Possibly, but for another PR.